test/unit/test_uop_symbolic.py

```diff
@@ -30,7 +30,8 @@ def Variable(expr, nmin, nmax):
   class TempVar:
     def __init__(self, x): self.expr = x
   #return UOp(UOps.DEFINE_VAR, dtypes.int, (UOp.const(dtypes.int, nmin), UOp.const(dtypes.int, nmax)), TempVar(expr))
-  return UOp(UOps.DEFINE_VAR, dtypes.int, tuple(), TempVar(expr))
+  return UOp(UOps.DEFINE_VAR, dtypes.int, (NumNode(nmin), NumNode(nmax)), TempVar(expr))
```

Comment:
i think this should use UOp.const instead of NumNode (the goal is to use uop to replace all symbolic for indexing, except symbolic shape)

Reply:
Yea, this isn't valid as it stands